src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -75,6 +130,12 @@ public void SetGssContext(SafeGssContextHandle context)
             _context = context;
         }
 
+        public void SetAuthenticationPackage(int isNtlm)
```

Comment:
ohk changed it to bool , Caller will convert int to bool first and then call this.